### DISCUSSION TITLE
Current master doesn't compile

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1168] [Team Posts]
I get a lot of syntax errors because of non existent constants like Qt::endl, Qt::dec ...

**Comment by spamatica** [comment]
Ah, I saw Tim mentioned a lot of cleanups after updating his system.
Missing include or is it some new qt feature that snuck in? 

Hmm, searched in the documentation and it seems it's new in qt5.14
https://doc-snapshots.qt.io/qt5-5.14/qt.html

no mention in the previous documentation
https://doc-snapshots.qt.io/qt5-5.12/qt.html

**Comment by kybos** [comment]
Yes, seems to be new, though it's not explicitly mentioned in the documentation (as is usually the case with non-backwards-compatible functions).

**Comment by terminator356** [comment]
Fixed in master now. Sorry for the trouble. Try it out....
Crud. I could swear I researched it.
Qt seems to get pretty lax sometimes in the docs, they usually say "this functions was added in Qt 5.xx"
 but not here. I hate when they do that, makes a lot of research.

**Comment by kybos** [comment]
It's working now, thanks!
